### ISSUE TITLE
allow copying alt text from images

### ISSUE THREAD

**Issue by igalic** [tuskyapp/Tusky#2126]
It's possible to fold out alt text from images, but it's not real text as it cannot be copied.

When following people who speak (many) different languages, it's convenient to copy the things they say to paste them into a translation engine

* * * *
- Tusky Version: 14.0
- Android Version: 10
- Android Device: One Plus 5t
- Mastodon instance (if applicable): cathode.church

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by stopregionblocking** [comment]
Came to open this same issue and almost submitted before searching again. Had used the language of "media descriptions" instead of alt text.

Tusky Version 19
Android Version 8

**Comment by jsoref** [comment]
I haven't discovered how to expand alt text. As alt text can be considerably longer than post bodies, a small info bubble isn't enough... 

Sample:
https://mastodon.social/@jsoref/109308434825452481
